code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -131,6 +131,19 @@
 			new /obj/item/weapon/grenade/plastic/c4 (src) // 1 tc
 			new /obj/item/weapon/card/emag(src) // 6 tc
 			return
+		
+		if("ninja") // 33 tc worth
+			new /obj/item/weapon/katana(src) // Unique , hard to tell how much tc this is worth. 8 tc?
+			new /obj/item/weapon/implanter/adrenalin(src) // 8 tc
+			new /obj/item/weapon/throwing_star(src) // 5 tc for all 5
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)

```

Comment:
I made that comment when there was 5 throwing stars and forgot to change it later, whooops